src/xmllang/graphml.cpp

```
@@ -186,10 +186,9 @@ bool read_graphml(
   graphmlt &dest,
   graphmlt::node_indext &entry)
 {
-  null_message_handlert message_handler;
   xmlt xml;
 
-  if(parse_xml(is, "", message_handler, xml))
+  if(parse_xml(is, "", null_message_handler, xml))
```

This is probably my code ... discarding error messages never was a good idea.